lib/analytical/modules/google.rb

```diff
@@ -19,6 +19,10 @@ def init_javascript(location)
             })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
             
             ga('create', '#{options[:key]}', 'auto');#{linkid}
+            var match = document.cookie.match(new RegExp('(^| )ignored_referrer=([^;]+)'));
+            if (match && decodeURIComponent(match[2]) === document.referrer) {
+              ga('set', 'referrer', document.location.origin);
```

Comment:
Minimum version for `document.location.origin` is IE11, but is not work in IE 11 (known bug)